src/Compiler/SyntaxTree/PrettyNaming.fs

```diff
@@ -1097,15 +1097,20 @@ let GetLongNameFromString x = SplitNamesForILPath x
 // Resource format for pickled data
 //--------------------------------------------------------------------------
 
+// Uncompressed OptimizationData/SignatureData name for embedded resource
 let FSharpOptimizationDataResourceName = "FSharpOptimizationData."
-
 let FSharpSignatureDataResourceName = "FSharpSignatureData."
 
+// Compressed OptimizationData/SignatureData name for embedded resource
+let FSharpOptimizationCompressDataResourceName = "FSharpOptimizationCompressData."
+let FSharpSignatureCompressDataResourceName = "FSharpSignatureCompressData."
```

Comment:
Since it describing the data, would `Compressed` be a better word than `Compress` here?

Reply:
Probably, I can certainly change that.

Reply:
done